### ISSUE TITLE
Command palette focused editor on close instead of previously focused item

### ISSUE THREAD

**Issue by setaskin** [microsoft/vscode#114590]
- VSCode Version:
- OS Version: 

Steps to Reproduce:

1. Go to VS Code explorer viewlet
2. tab through "Outline" section's "collapse all" button via keyboard. 
3. click on CTRL+ Shift + P to open command line 
4. click on  ESC button 

Expected : focus should go back to  "collapse all" button .
Actual : focus goes back to the editor. 

![FocusBugFinal](https://user-images.githubusercontent.com/37163639/105084878-d35bfc00-5a4b-11eb-96c3-8094d3a0c0ea.gif)

**Comment by setaskin** [comment]
@isidorn this is the issue we discussed before. We see the same behavior on Live Share extension but I think the root cause is coming from VS Code.

**Comment by isidorn** [comment]
@setaskin thanks for creating this issue. Yes, this is how VS Code behaves, when closing things we do not always respect the previous focus. So in this case the quick pick on close simply focused the editor.

I am not sure if we would change this since we did not recieve any feedback from our accessibility community that this would be ideal behavior.
For now leaving this open to get more feedback

fyi @chmarti @saschacowley

**Comment by yume-chan** [comment]
prior discussions #44722 #46961

**Comment by setaskin** [comment]
Thanks everyone!. We have an accessibility bug on ADO  for this issue. Since we have no code to manipulate the focus order, I will comment on the bug to transfer it to VS Code side.

**Comment by isidorn** [comment]
Sure we can reopen this to gather more feedback, but we have no plans to change this.

**Comment by Shivamd1** [comment]
@isidorn Could you please let us know when this will be fixed? This bug needs to be fixed as this is affecting compliance of another product.

**Comment by isidorn** [comment]
No plans to fix.

**Comment by Shivamd1** [comment]
@isidorn This bug is logged as external which have severity of Sev2 so this bug can't be closed.
Also if you need an exemption or lowering of severity please join office hours

**Comment by isidorn** [comment]
@Shivamd1 yeah lowering the severity sounds like a good path forward. I do not understand what does "join office hours" means. I am in CET timezone.

**Comment by Shivamd1** [comment]
@isidorn Could you please provide any update on this?

**Comment by isidorn** [comment]
No updates

**Comment by setaskin** [comment]
Hi @isidorn , the accessibility team wants us to give fix date estimate on this ? Do you have an approx.. estimate when would it be fixed ?

**Comment by fsteffi** [comment]
**GitHubTags**:#A11yMAS;#A11yTCS;#Win32;#DesktopApp;#A11y_VSCode;#WCAG2.4.3;#A11ySev2;#VisualStudioCode-Win32-Apr2021;#Benchmark;#Visual Studio Code Client;#Keyboard;

**Comment by isidorn** [comment]
Approx estimate: next 3 months.

**Comment by TylerLeonhardt** [comment]
verification steps:

* click on Outline and make sure it's expanded:
![image](https://user-images.githubusercontent.com/2644648/140826783-0f29b677-26d2-43da-9e79-623a997932dd.png)
* tab over to the icon button:
![image](https://user-images.githubusercontent.com/2644648/140826821-42bb3d8f-3408-45bf-9001-0c0a71c4d379.png)
* open the command palette and then hit ESC

if the icon button was still visible, then the button should be focused
if the icon button was not visible, then the Outline title bar should be focused

make sure you've at least done it once where the button has disappeared as that is what this issue is talking about.

**Comment by isidorn** [comment]
Works amazing -> verified. Thanks a lot 👏
